src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -265,6 +275,15 @@ export default function ChartExpandedState({ asset }) {
     ChartExpandedStateSheetHeight -= 60;
   }
 
+  const { navigate } = useNavigation();
+
+  const handleL2ExplainerPress = useCallback(() => {
+    navigate(Routes.EXPLAIN_SHEET, {
+      network: assetWithPrice.type,
+      type: assetWithPrice.type,
```

Comment:
you're using the same `type` key for both, maybe on purpose, maybe not

Reply:
same here: `network` not needed here currently (not used in route params in explain sheet, only the type is used)